### PR TITLE
Content for Homepage 

### PR DESCRIPTION
**Pull request by LavishRawat**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (Abhinegi2) [inline src/App.css:1]: remove unnecessary spaces

File: src/Components/Homepage/homepage.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.align {
+    text-align: center;
+    margin-top: 185px;
+}
+
```
Comment (Abhinegi2) [inline src/Components/Homepage/homepage.css:5]: remove spacing

File: src/Components/Navbar/navbar.css [hunk 3]
```diff
@@ -1,5 +1,5 @@
 .navbar {
-    width: 100%;
+    width: auto;
     margin-left: 10pc;
```
Comment (Abhinegi2) [inline src/Components/Navbar/navbar.css:3]: can we use px!

File: src/Components/Navbar/navbar.css [hunk 4]
```diff
@@ -1,5 +1,5 @@
 .navbar {
-    width: 100%;
+    width: auto;
```
Comment (rohit123-rawat) [inline src/Components/Navbar/navbar.css:2]: reset this one to 100px

**Pull request merged**